lib-ziti-tunnel-cbs/ziti_hosting.c

```diff
@@ -35,7 +35,7 @@ static void ziti_conn_close_cb(ziti_connection zc) {
     struct hosted_io_ctx_s *io_ctx = ziti_conn_data(zc);
     if (io_ctx) {
         ZITI_LOG(TRACE, "hosted_service[%s] client[%s] ziti_conn[%p] closed",
-                 io_ctx->service->service_name, ziti_conn_source_identity(zc), zc);
```

Comment:
It's not a big deal, but I see a lot of these single-space changes which look accidental?